app/feature/feature-odyssey/src/main/kotlin/com/hedvig/android/feature/odyssey/step/fileupload/FileUploadViewModel.kt

```diff
@@ -77,6 +77,9 @@ internal class FileUploadViewModel(
   }
 
   fun addLocalFile(uri: Uri) {
+    if (uri.toString() in _uiState.value.localFiles.map { it.id }) {
```

Comment:
Great, I've just met this bug today :)

Reply:
Hehe yeah it's a very easy thing to miss, and in our case I figured out because it showed as a crash in crashlytics/ddog, because the lazy list was using this ID as a unique key which was no longer unique so compose just crashes in those scenarios.
I am glad we found the underlying issue though :D